README.md

```diff
@@ -75,6 +75,37 @@ If you use `MESH_LOWPAN_ND` or `MESH_THREAD` you need to specify your radio modu
     }
 ```
 
+If you use `CELLULAR` you must specify the following:
+
+```json
+    "target_overrides": {
+        "*": {
+            "ppp-cell-iface.apn-lookup": true
+        }
+```
```

Comment:
in-balanced {} pair, making a lot of red text underneath it. Please add the balancing }.

Reply:
Ah, I wondered what the red was all about.  Will do.